### ISSUE TITLE
JCheckBox and JRadioButton text missing

### ISSUE THREAD

**Issue by Stephane-D** [Insubstantial/insubstantial#103]
I updated to last insubstancial and if the text AA problem is indeed fixed we have now missing text for JCheckBox and JRadioButton components (some others may be affected too).

**Comment by shemnon** [comment]
Before and after pictures please.

On Mon, Nov 5, 2012 at 3:14 AM, Stephane Dallongeville <
notifications@github.com> wrote:

> I updated to last insubstancial and if the text AA problem is indeed fixed
> we have now missing text for JCheckBox and JRadioButton components (some
> others may be affected too).
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/Insubstantial/insubstantial/issues/103.

## 

There is nothing that will hold me back.  I know who I am....
I remember wher I came from, and I feel stronger for knowing.
Zane, Ninja of Ice.  Ninjago S01E07

**Comment by Stephane-D** [comment]
Here are the images :)

**Before**
![Before](http://images4.hiboox.com/images/4512/3a5d2f15c8fcb6d8e3c6deb925a2efe7.jpg)

**After**
![After](http://images4.hiboox.com/images/4512/222d5c5d680c09339254a0027c631173.jpg)

**Comment by shemnon** [comment]
Give it a twirl with the 7.2.1 snapshot, add this to your pom

```
<repositories>
  <repository>
    <id>snapshots-oss.sonatype.org</id>
    <name>Sonatype Snapshots Repository</name>
    <url>http://oss.sonatype.org/content/repositories/snapshots/<;/url>
    <releases>
      <enabled>false</enabled>
    </releases>
    <snapshots>
      <enabled>true</enabled>
    </snapshots>
  </repository>
<repositories>
```

and then depend on the `7.2.1-SNAPSHOT` version

**Comment by enwired** [comment]
I don't know whether it works for everyone, but I tested 7.2.1-SNAPSHOT and it worked great for me!  The text is anti-aliased in the way I expect.  Thanks!

PS: There are two little typos in your pom snippet above: extra character in "<;/url>" and the final line should have "/repositories".

**Comment by Stephane-D** [comment]
Yeah the problem is fixed in the 7.2.1 snapshot version :)
